flips/20201125-bip-44-multi-account.md

```diff
@@ -0,0 +1,196 @@
+# Application of BIP 44 in Flow Wallets
+
+| Status        | Proposed |
+:-------------- |:---------|
+| **FLIP #**    | [200](https://github.com/onflow/flow/pull/200) |
+| **Author(s)** | Peter Siemens (peter@dapperlabs.com), Jeffery Doyle (jeffrey.doyle@dapperlabs.com) |
+| **Sponsor**   | Peter Siemens (peter@dapperlabs.com) |
+| **Updated**   | 2020-11-25 |
+
+## Objective
+
+This document aims to define a standard pattern for the application of 
+[Bitcoin Improvement Proposal 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) (BIP 44)
+in Flow-compatible hardware and software wallets.
+
+## Motivation
+
+This proposal was prompted by the recent development of the first Flow hardware
+wallet application (Ledger) and its companion host software (Flow Port). 
+
+In the near term, this proposal affects users of the Flow Ledger application.
+In the long term, it affects Flow wallet developers and their users.
+
+## User Benefit
+
+By following this standard, developers of existing BIP 44 crypto wallets will
+be able to reduce friction for users who wish to create accounts on Flow.
+
+## Design Proposal
+
+### Path Levels
+
+BIP 44 defines an implementation of 
+[Bitcoin Improvement Proposal 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (BIP 32)
+that standardizes a format for hierarchical deterministic (HD) 
+key derivation paths that support multiple coins and multiple accounts per coin.
+
+The BIP 44 path format is as follows:
+
+```
+m / purpose' / coin_type' / account' / change / address_index
+```
+
+When using this format to construct or discover a Flow user wallet, 
+the value(s) of each path parameter should follow the specifications below.
+
+#### Purpose
+
+Purpose is always the constant 44' (0x8000002C), 
+as described in [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#purpose).
+
+#### Coin Type
+
+Coin type is always the constant 539' (0x8000021b), 
+the unique identifier for the Flow protocol, as defined in 
+[SatoshiLabs Improvement Proposal 44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) (SLIP 44).
+
+#### Account
+
+Account is the index of a distinct account address owned by the wallet user. 
```

Comment:
This index also has no relation to the `Address state index` used to generate the Flow Addresses, correct? Is that worth noting? It's something i keep thinking about, but am likely an outlier.

Reply:
Great point. There's no relation, I'll make that more clear